lib/rack/lint.rb

```diff
@@ -655,6 +657,29 @@ def check_hijack_response(headers, env)
     ## * Middleware should not wrap the IO object for the request pattern. The
     ##   request pattern is intended to provide the hijacker with "raw tcp".
 
+    ## === Early Hints
+    ##
+    ## The application or any middleware may call the <tt>rack.early_hints</tt>
+    ## with pairs of headers that will be sent as a 103 Early Hints provisional
+    ## response.
+    def check_early_hints(env)
+      if env[RACK_EARLY_HINTS]
+        ##
+        ## If rack.early_hints is present it must respond to #call.
+        assert("rack.early_hints must respond to call") { env[RACK_EARLY_HINTS].respond_to?(:call) }
+        original_callback = env[RACK_EARLY_HINTS]
```

Comment:
This local variable is not used.  It looks like other than checking `env['rack.early_hints']` responds to call, the value is ignored.  Shouldn't `original_callback` be called with the headers inside the lambda below?  Otherwise `Rack::Lint` breaks applications using early hints?